src/main/java/frc/robot/Constants.java

```diff
@@ -3,8 +3,8 @@
 
 public final class Constants {
 
-    public static class ExampleSubsystem {
-        public static final double POWER = 0.5; // [%]
+    public static class Hood {
+        public static final double CLOSED_ANGLE = 54; // Angle for longer distance. [deg]
+        public static final double OPEN_ANGLE = 69; // Angle for shorter distance. [deg]
```

Comment:
These aren't used anywhere.
```suggestion
```

Reply:
They may be used for trajectory calculations. It's better to leave them there so that we don't forget.

Reply:
> It's better to leave them there so that we don't forget.

That's bad practice. Leaving unused code is bad and will clutter your codebase.
This code is committed. You can safely delete it and revert the deletion if needed in the future.